docs/mobile-apps/automated-testing/espresso-xcuitest/espresso-capture.md

```diff
@@ -0,0 +1,39 @@
+---
+id: espresso-capture
+title: Espresso Screenshot Capture
+sidebar_label: Espresso Screenshot Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Espresso Screenshot Capture functionality allows you to easily take a snapshot of the error as to support with visual verifications of your debugging process. These screenshots also help in identifying any errors or bugs within your application.
+
+
+## Capturing Screenshots
+
+To take screenshots during an Espresso test run, Sauce Labs provides the code you can add to your Espresso tests. See our [sample app repository](https://github.com/saucelabs/my-demo-app-android/blob/1d2d85aad21bc76878d6aa4d77aefd889b59d3c9/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java) and copy the utility class into your project. Make sure to handle exceptions in your test code to ensure a proper test run. This feature is supported on Android 4.3 and above.
+
+#### Perform the following steps to edit your scripts and capture screenshots:
+
+1. Add the following class to your project. (See our Java an example [here](https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java)).
```

Comment:
Is there a specific code sample we can provide for the first step? If not, the first step should be changed to:

```suggestion
1. Add the utility class to your project. See our Java example on [GitHub](https://github.com/saucelabs/my-demo-app-android/blob/2daaab68f6b75dcd78533dda7ac1715eec070f99/app/src/androidTest/java/com/saucelabs/mydemoapp/android/screenshot/SauceLabsCustomScreenshot.java).
```